src/Evolution/Ringdown/StrahlkorperCoefsInRingdownDistortedFrame.hpp

```diff
@@ -0,0 +1,37 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataVector.hpp"
+
+#include <array>
+#include <cstddef>
+#include <string>
+#include <vector>
+
+/// \brief Transform `Strahlkorper` coefs to ringdown distorted frame.
+///
+/// \details Reads Strahlkorper coefficients (assumed to be in the inertial
```

Comment:
These docs should go on the function, not the namespace. But also probably good to document the namespace as well